src/components/Header/Header.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useContext, useState } from 'react';
+import { DispatchContext, StateContext } from '../../context/ToDoContext';
+import classNames from 'classnames';
+
+export const Header: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const { todos } = useContext(StateContext);
+  const dispatch = useContext(DispatchContext);
+
+  const handleCreateNew = () => {
+    dispatch({
+      type: 'createNew',
+      newTodo: {
+        id: +new Date(),
+        title: title,
+        completed: false,
+      }
+    });
+    setTitle('');
+  };
+
+  const handleKeyPress = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (!title) {
+      return;
+    }
+
+    if (event.key === 'Enter') {
+      event.preventDefault();
+      handleCreateNew();
+    }
+  };
+
+  const handleToggle = () => {
+    dispatch({type: 'toggleStatus'});
+  };
+
+  return (
+    <header className="todoapp__header">
+      {todos.length !== 0 && (
+        <button
+          type="button"
+          className={classNames(
+            "todoapp__toggle-all",
+            {'active': todos.every(plan => plan.completed)},
```

Comment:
Move this conditional for a variable

Reply:
still not resolved
![image](https://github.com/mate-academy/react_todo-app/assets/55200530/ba2b5302-db10-4d0f-96a7-adf52dce3a19)

Reply:
still not fixed
<img width="628" alt="image" src="https://github.com/mate-academy/react_todo-app/assets/70220667/f36aea11-e4ed-45d5-a570-b37497949668">